### ISSUE TITLE
mumble and mumble-server aren't found in repo (Ubuntu)

### ISSUE THREAD

**Issue by machtyn** [mumble-voip/mumble#3500]
I followed the instructions of adding the ppa:/mumble/release, updated, and then tried to install. But I keep getting the error that the package is not found.

![image](https://user-images.githubusercontent.com/16729235/44567390-31cc0580-a740-11e8-90fb-71eddbc322e8.png)

My plan is to use this locally with my family as some computers are in different rooms.  And, yes, I'm installing the server on a virtualbox VM.

![image](https://user-images.githubusercontent.com/16729235/44567476-912a1580-a740-11e8-911b-b5741430055d.png)

**Comment by davidebeatrici** [comment]
I would add `mumble-snapshot`, as the latest builds failed in `mumble-release`:
```
add-apt-repository ppa:mumble/snapshot
apt update
apt install mumble-server
```

**Comment by machtyn** [comment]
I looked through the open ubuntu reports, but didn't see anything specifically stating this was a known issue. Perhaps I missed it.  Thanks, I'll use snapshot.

**Comment by davidebeatrici** [comment]
You're welcome, we wasn't aware of the issue.

**Comment by Kissaki** [comment]
Fixed in #3689, 1.3.0 is not available in our release and snapshot PPA for all current Ubuntu versions.

If you are still experiencing problems please create a new issue.
